### PR TITLE
Disable Angular development mode when NODE_ENV !== 'development'

### PR DESCRIPTION
**Pull request by xorgy**
This addresses #530

### CODE REVIEW

**Comment by sumitarora** [comment]
@xorgy Setting the prod env should be part of the prod build also? Like `"build": "cross-env NODE_ENV=production webpack",` ?

**Comment by xorgy** [comment]
Setting NODE_ENV would need to be part of the build environment, yeah. I wanted to get input on where/whether to do that in this PR.

**Comment by sumitarora** [comment]
@xorgy Any progress on setting node env. as part of build using `package.json`

**Comment by xorgy** [comment]
@sumitarora Sorry for the delay, forgot to push this yesterday.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
   },
   "homepage": "https://github.com/rangle/augury",
   "scripts": {
-    "build": "npm run webpack --colors --display-error-details --display-cached",
+    "build": "webpack --colors --display-error-details --display-cached",
```
Comment (igor-ka) [inline package.json]: @xorgy how come no `npm run` anymore prepending the `webpack`?

**Comment by sumitarora** [comment]
@xorgy Looks good one last change to run in watch mode we use `npm start` please configure it to run in dev mode also. After that change 🚢 💯

File: package.json (hunk 1 above)
Comment (xorgy) [inline package.json]: If just "webpack" is safe for `npm run webpack`, then "webpack --colors --display-error-details --display-cached" is safe for `npm run build`

**Comment by xorgy** [comment]
@sumitarora Will-do.

File: package.json (hunk 1 above)
Comment (igor-ka) [inline package.json]: Sounds good @xorgy thanks for the explanation

**Comment by igor-ka** [comment]
@xorgy LGTM :shipit:

**Pull request merged**